### DISCUSSION TITLE
[BUG] Cuando uno va a contactos para conversar con una persona, no se crea/abre el canal de conversacion

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#148] [4. Bug Reports] [answered]
Hola
Un cliente me reportó que al ir a la "**lista de contactos**" para conversar con un usuario, no puede conversar. Sin embargo, si voy a Mensaje directos (o private), selecciono un canal, luego voy a "**lista de contactos**", selecciono una persona me aparecerá el canal de conversación

Adjunto video.
Link: https://drive.google.com/file/d/1P5he5JdDkPJBlwJcpWrP3MGxl6NP8p9g/view?usp=sharing

**Edit:** Agrego más información
Cuando el usuario hace por primera vez el inicio de sesión y se va a la lista de mis contactos, no le aparecerá. Pero si va a un canal de conversación y regresa a la lista de mis contactos, saldrá el canal.
Link: https://drive.google.com/file/d/18zxyzqlcGPwx-BXafNKMZNVOfuGLO1nv/view?usp=sharing

Gracias

**Comment by acedward** [comment, marked as answer]
Tenemos un bug aquí. Estamos revisando

**Reply by NicolasPLG** [reply]
gracias @acedward

**Reply by acedward** [reply]
El bug está categorizado de alta prioridad.
Deberíamos tener noticias pronto

**Reply by NicolasPLG** [reply]
gracias @acedward por tu pronta respuesta
